src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -161,16 +163,21 @@ private async Task ExecuteAsync(HttpContext context, ISchemaFactory schemaServic
                     FieldImpact = _settings.FieldImpact
                 };
                 _.Listeners.Add(dataLoaderDocumentListener);
+                _.RequestServices = context.RequestServices;
+                _.ThrowOnUnhandledException = true;
```

Comment:
this setting should ensure that exceptions are thrown rather than wrapped in result - just for debugging, should not be in final version, will remove.